.circleci/config.yml

```diff
@@ -57,9 +57,9 @@ aliases:
   - &conda_upload
     name: conda_upload
     command: |
-       if [[ $CIRCLE_BRANCH != "master" ]]; then
-          exit 0
-       fi
+       #if [[ $CIRCLE_BRANCH != "master" ]]; then
```

Comment:
Revert changes in this file before merging.

Reply:
Looks like you need to revert these changes?